### PR TITLE
Update toolbarbutton.css

### PR DESCRIPTION
**Pull request by PimpUigi**
"CTR" id updated to "CTRaddon"

### CODE REVIEW

File: toolbarbutton.css [hunk 1]
```diff
@@ -22,7 +22,7 @@
 	}
 
 	/*start - support for classic theme restorer addon*/
-	toolbar[id="ctr_addon-bar"] > #navigator-throbber {
+	toolbar[id="ctraddon_addon-bar"] > #navigator-throbber {
```
Comment (Noitidart) [inline toolbarbutton.css:25]: hey man this wont support the older versions of ctr_addon so please make new commit, please make the title of the commit be `1.1.rev15` and also change this line to be `toolbar[id="ctr_ddon-bar"] > #navigator-throbber, toolbar[id="ctr_addon-bar"] > #navigator-throbber {`

I seriously appreciate your work and commitment to this man. I would really appreciate if you could use the same naming scheme I do on the commits :) so i can merge you in and not copy your merge. i want your name to be on there :)

thanks man so much!

Comment (Noitidart) [inline toolbarbutton.css:25]: you'll have to re edit, otherwise it will try to make me merge two commits :( forgive me man im so picky haha

Comment (PimpUigi) [inline toolbarbutton.css:25]: Alright. I um, can take a guess at how it goes.

I don't need my name on there though. That's not really important to me. : )

Michael K. Pessolano

Comment (Noitidart) [inline toolbarbutton.css:25]: I want your name I really love it when people collaborate :)
When i moved from the -1 repo to this repo it ripped my heart out to have to copy your commit :(

**Pull request closed without merging**